getting-started/installation-apt-debian.md

```diff
@@ -2,11 +2,11 @@
 
 This will install TimescaleDB via `apt` on Debian distros.
 
-**Note: TimescaleDB requires PostgreSQL 9.6.3+, 10.2+, or 11.0+**
+**Note: TimescaleDB requires PostgreSQL 9.6.3+, 10.9+, or 11.4+**
```

Comment:
why is 9.6 not affected? did you adjust package metadata as well?

Reply:
I did not update the version I built against for 9.6 (not sure why), so as far as I know, it should still work back to 9.6.3 on platforms with that version